module-db/Interface/ContactRecord.cpp

```diff
@@ -595,6 +598,9 @@ auto ContactRecordInterface::updateQuery(const std::shared_ptr<db::Query> &query
     auto ret         = ContactRecordInterface::Update(updateQuery->rec);
     auto response    = std::make_unique<db::query::ContactUpdateResult>(ret);
     response->setRequestQuery(query);
+    if (ret) {
```

Comment:
minor suggestion: as you are at it, could you unify naming of functions returns to result in this interface? it is either ret or result :smile:

Reply:
Ok, I'll unify this